builder/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM debian:sid
+
+ENV NODE_ENV=production
+
+# Install python and dependencies to download NodeJS
+RUN apt-get update && apt-get install -y -q \
+    bc \
+    curl \
+    libelf-dev \
+    software-properties-common \
+    tree \
+    python3 \
+    python3-pip
+
+# Download and install NodeJS 10
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN apt-get install -y nodejs
+
+RUN mkdir /app
+WORKDIR /app
+
+# Install nbuild dependencies
```

Comment:
Ideally you want to add the `git clone/git pull nbuild` in the Dockerfile and do not require any user interaction.

This is because i'm counting on Docker Hub to automatically build our Docker images, and they obviously don't parse the README for any shell command to run beforehand :p

Reply:
The image is pulled from docker hub directly